x11-misc/yabar/metadata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>lillyjsm@gmail.com</email>
+		<name>Jake Lilly</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">Yabar is a modern and lightweight status bar that is intended to be used along with minimal X window managers.</longdescription>
+	<use>
+		<flag name="playerctl">Uses <pkg>media-sound/playerctl</pkg> to retrieve the name of the currently played song.</flag>
```

Comment:
No dot in the end here please, to try and follow common style.

Reply:
removed